sh/test.sh

```
@@ -16,16 +16,16 @@ function endfold {
   fi
 }
 
-# run integration tests
-startfold "Running Integration Tests..." "test-integration";
-./sh/integration-tests.sh
-endfold "test-integration" ;
-
 # run karma (client unit) tests
 startfold "Running Client Unit Tests..." "test-client-unit";
 ./node_modules/.bin/karma start --single-run --no-auto-watch --concurrency 1 karma.conf.js
 endfold "test-client-unit" ;
 
+# run integration tests
+startfold "Running Integration Tests..." "test-integration";
+./sh/integration-tests.sh
+endfold "test-integration" ;
+
```

by curiosity, is there any reason why the integration test must run after the client unit test (karma) ?

---

In general, unit tests tend to be faster, so I moved up the unit tests as far as I could so they "fail fast" and give us an error as soon as possible if there is a unit test error.